examples/common/async_asyncio_client.py

```diff
@@ -204,12 +201,12 @@ def run_with_no_loop():
     # Run with No loop
     log.debug("Running Async client")
     log.debug("------------------------------------------------------")
-    run_with_no_loop()
+    # run_with_no_loop()
```

Comment:
Why was this outcommented? Is it needed anymore?

Reply:
look at the file, it is one of different options.